src/index.js

```diff
@@ -111,17 +111,30 @@ class EventsCoreAPI {
    * =========================================================================
    */
 
+  /**
+   * @typedef {object} ProviderOptions
+   * @property {string} [providerMetadataId] Fetch by providerMetadataId for the consumer org
+   * @property {string} [instanceId] For Self registered providers, instanceId is a must while fetching by providerMetadataId
+   * @property {Array} [providerMetadataIds] Fetch all providers ( and all instances ) for the list of provider metadata ids
```

Comment:
Can we add the type of the element of the array, I see in the generated docs it is an array of any, I think we can specify with `Array.<string>`, but I am not 100% sure

Reply:
done